tpp-app/src/services/utils/models.js

```diff
@@ -1,7 +1,7 @@
 /**
  * The symptoms of each day
  */
-class Symptoms {
+export class Symptoms {
```

Comment:
sorry this was unclear can we add like an exercise_minutes field to this and store amount of user's exercise in minutes? 
<img width="581" alt="Screen Shot 2022-02-28 at 9 30 35 PM" src="https://user-images.githubusercontent.com/35851484/156093321-d298f5db-6e25-47a4-a55d-e4f85f6a29ea.png">

Reply:
Added new `ExerciseActivity` object in `models.js`! Which will be assigned to `Symptoms.exercise`